packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1051,7 +1051,7 @@ class BoxDecoration extends Decoration {
   /// * If [backgroundColor] is null, this decoration does not paint a background color.
   /// * If [backgroundImage] is null, this decoration does not paint a background image.
   /// * If [border] is null, this decoration does not paint a border.
-  /// * If [borderRadius] is null, this decoration use more efficient background
+  /// * If [borderRadius] is null, this decoration uses more efficient background
```

Comment:
I added support for clipping the backgroundImage when borderRadius is specified.

![screenshot](https://cloud.githubusercontent.com/assets/1377460/21334253/bf820884-c60a-11e6-930b-01dd20fb4918.png)